specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2017-10-01/containerregistry.json

```diff
@@ -1014,12 +1055,76 @@
     }
   },
   "definitions": {
+    "ImportImageParameters": {
+      "required": [
+        "sourceRepository",
+        "source"
+      ],
+      "type": "object",
+      "properties": {
+        "sourceRepository": {
+          "description": "Repository name of the source image.",
+          "type": "string"
+        },
+        "sourceTag": {
+          "description": "The tag name of the source image.  When both source tag and source manifest are omitted the 'latest' tag will be used.\r\nExclusive with SourceManifestDigest.",
+          "type": "string"
+        },
+        "sourceManifestDigest": {
+          "description": "The manifest sha of the source image. Exclusive with SourceTag.",
+          "type": "string"
+        },
+        "source": {
+          "$ref": "#/definitions/ImportSource",
+          "description": "The source of the image."
+        },
+        "targetTags": {
+          "description": "List of strings of the form repo[:tag].  When tag is omitted the source will be used (or 'latest' if source tag is also omitted.)",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "untaggedTargetRepositories": {
+          "description": "List of strings of repository names to do a manifest only copy.  No tag will be created.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "mode": {
+          "description": "When Force, any existing target tags will be overwritten.  When NoForce, any existing target tags will fail the operation before any copying begins.\r\nNoForce is the default.",
```

Comment:
Can we add something like `default: NoForce` rather than mentioning the default value in description?
NIT: remove the extra space in `overwritten.  When`. There are extra spaces in other descriptions too.

Reply:
done